deployment/serverless.yml

```
@@ -50,9 +58,36 @@ plugins:
 #           - 'uec-pu-apu-2429-rds'
 
 functions:
+  event-receiver:
+    image: ${env:AWS_SAME_ACCOUNT_DOCKER_REGISTRY}/event-receiver:latest
```

Can the version for both event-receiver and event-sender be set to ${env:VERSION}. This is so multiple people can deploy and we can deploy non latest versions
```suggestion
    image: ${env:AWS_SAME_ACCOUNT_DOCKER_REGISTRY}/event-receiver:${env:VERSION}
```

---

updated